src/cryptoadvance/spectrum/spectrum.py

```diff
@@ -542,6 +542,13 @@ def testmempoolaccept(self, rawtxs, maxfeerate=0.1):
         return [{"allowed": True} for tx in rawtxs]
 
     @rpc
+    def getrawtransaction(self, txid, verbose=False):
```

Comment:
Can you maybe add descriptions to the method? I had a short discussion with chatGPT and after some back and forth, i liked this version:
![image](https://user-images.githubusercontent.com/117085/213401656-fa8d2025-6a85-4e3f-9981-d219806b319e.png)

```suggestion
    def getrawtransaction(self, txid, verbose=False):
    """
    Get raw transaction data for a given transaction id.
    For more information on the Bitcoin RPC call see: https://developer.bitcoin.org/reference/rpc/getrawtransaction.html
    
    Parameters:
    - txid (str): The transaction id of the transaction you want to retrieve.
    - verbose (bool): Indicates whether to return detailed information about the transaction. Default is False.
    
    Returns:
    - dict: If verbose is set to True, it returns detailed information about the transaction specified by txid,
        otherwise it returns only the transaction data.
    Implementation details:
    - This method is using the ElectrumX API call `blockchain.transaction.get` which is documented here:
        https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-transaction-get
    """
```
Doing such a discussion for other API calls is a great way of learning about the Bitcoin API as well as the Electrum API. So maybe you're interested doing that?

Reply:
Good point, but it's a bit much, I'd leave this part out, this can be checked better (examples etc.) via the two links anyways:

```
    Parameters:
    - txid (str): The transaction id of the transaction you want to retrieve.
    - verbose (bool): Indicates whether to return detailed information about the transaction. Default is False.
    
    Returns:
    - dict: If verbose is set to True, it returns detailed information about the transaction specified by txid,
        otherwise it returns only the transaction data.
```

Reply:
Never thought of this way to learn about anything. Will definitely have a chat with chatgpt on it😄
Should I implement the documentation changes you suggested above @k9ert?